build/cmake/lib/nanosvg.cmake

```diff
@@ -0,0 +1,41 @@
+#############################################################################
+# Name:        build/cmake/lib/nanosvg.cmake
+# Purpose:     Use external or internal nanosvg lib
+# Author:      Tamas Meszaros, Maarten Bent
+# Created:     2022-05-05
+# Copyright:   (c) 2022 wxWidgets development team
+# Licence:     wxWindows licence
+#############################################################################
+
+if(wxUSE_NANOSVG STREQUAL "builtin")
+    set(wxUSE_NANOSVG_EXTERNAL 0 PARENT_SCOPE)
+elseif(wxUSE_NANOSVG)
+    set(wxUSE_NANOSVG_EXTERNAL 1 PARENT_SCOPE)
+
+    set(NANOSVG_LIBRARIES )
+    set(NANOSVG_INCLUDE_DIRS )
+
+    find_package(NanoSVG REQUIRED)
+
+    set(wxUSE_NANOSVG_EXTERNAL_DISABLE_IMPL FALSE)
+
+    foreach(TARGETNAME NanoSVG::nanosvg NanoSVG::nanosvgrast unofficial::nanosvg)
+        if(NOT TARGETNAME)
```

Comment:
This line fails for me but works if I change it to `if (NOT TARGET TARGETNAME)`

Reply:
This is indeed the correct syntax, I don't know why the other one works for me too. I'll update the PR.

Reply:
CMake can differ between versions, or who knows :)